contents/handbook/people/team-structure/teams-app-east-and-west.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Teams App East & West
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+[See team structure page](/handbook/people/team-structure/team-structure)
+
+## Overall goal for app and platform
+
+### Objective
+
+PostHog can handle 1bn events/day
+
+### Motivation
+
+The highest value users are likely to have high volumes. No-one in our enterprise pipeline has more volume than this.
+
+### Owner
+
+Karl
+
+### Key results
+
+- P95 <15 seconds for Suprdaily
+- Successful load test with Billion event queries
+
+## Sub team - App East
+
+### Objective
+
+Significantly increase the reliability, experience and completeness of our most frequently used tools
+
+### Motivation
+
+We believe that quality leads to more usage and more value from the product, focussing on quality today will drive more discoveries in the long term
+
+### Objective owner
+
+Marius
+
+### Key results
+
+- Shipped features to close competitive gaps in trends
```

Comment:
Do we have a list of them?

Reply:
From the offsite these items stand out:

- Really great insights experience. Wow!
- Export as PNG and share (add your logo)
- Subscribe via email to a dashboard or an insight
- Slack previews for insight urls